README.md

```diff
@@ -5,6 +5,7 @@ The documentation can be found [here](https://apidocs.microsoft.com/services/par
 ## Requirements.
 
 Python 3.7+ (Expected to work with Python 3.6+)
+Java JDK 8
```

Comment:
```suggestion
- Python 3.7+ (Expected to work with Python 3.6+)
- Java JDK 8
```

Reply:
Let's turn this into a list in markdown so that it doesn't do this.
<img width="397" alt="image" src="https://user-images.githubusercontent.com/9027725/174202533-c69f4bbe-023b-45c9-bb15-b39529e6fb48.png">